lib/Github/HttpClient/Cache/GaufretteCache.php

```diff
@@ -30,17 +30,17 @@ public function __construct(Filesystem $filesystem)
      */
     public function get($id)
     {
-        $content = $this->filesystem->read($id);
+        $content = ResponseSerializer::unserialize($this->filesystem->read($id));
```

Comment:
this is wrong. You should be replacing the call to ``unserialize`` below instead, as you are replacing the ``serialize`` call